Yosemite/Yosemite/Payments/CardReaderConnectionStateMachine.swift

```diff
@@ -1,13 +1,68 @@
 import Combine
 import Foundation
-import UIKit
 import Storage
 import SwiftUI
-import Yosemite
+
+public enum CardReaderConnectionResult {
```

Comment:
I know this is from before, but should we use a different name for this enum? `*Result` implies there's a result type to be expected, which is not the case, and later we find signatures like: 
```
private var onCompletion: ((Result<CardReaderConnectionResult, Error>) -> Void)?
```

Perhaps something like CardReaderConnectionStatus?